projects/core/src/lib/components/feature-info/state/feature-info.selectors.ts

```diff
@@ -135,3 +136,36 @@ export const selectFeatureInfoLayerListItems = createSelector(
     }));
   },
 );
+
+export const selectCurrentFeatureForEdit = createSelector(
+  selectFeatureInfoFeatures,
+  selectSelectedFeatureInfoLayer,
+  selectFeatureInfoMetadata,
+  (features, selectedLayer, featureInfoMetadata) => {
+    if (!selectedLayer || !selectedLayer.selectedFeatureId) {
+      return null;
+    }
+    const feature = features.find(f => f.__fid === selectedLayer.selectedFeatureId);
+    if (feature && featureInfoMetadata) {
+      const newAttributes = { ...feature.attributes, geom: feature.geometry };
+      const newFeature: FeatureInfoFeatureModel = {
+        ...feature,
+        attributes: newAttributes,
+      };
+      const filteredMetadata = featureInfoMetadata.filter(m => m.layerId === feature.layerId);
+      const newColumnMetadata = [
+        ...filteredMetadata,
+        {
+          layerId: feature.layerId,
+          key: 'geom',
```

Comment:
Dit werkt niet voor bijvoorbeeld de 'WATERDEEL' laag, bij editen krijg je niet de geometrie in beeld en ik verwacht ook dat bij opslaan de geometrie niet aangepast wordt, want niet bij elke laag heet het geometrie-attribuut `geom`. 
Je kan het geometrie-attribuut opzoeken in de columnMetadata, of zelfs in de feature info bij opvragen van attributen de `geometryInAttributes` parameter op true te zetten.